app/processing/statement/components/get-detailed-payments.js

```diff
@@ -2,16 +2,17 @@
 const moment = require('moment')
 const { convertToPounds } = require('../../../utility')
 
-const getDetailedPayments = async (calculation, paymentRequest, settlement) => {
+const getDetailedPayments = async (calculation, latestPayment, settlement) => {
   const payments = []
 
   const payment = {
     invoiceNumber: calculation.invoiceNumber,
     reference: settlement.reference,
-    dueDate: formatDate(paymentRequest.dueDate),
+    dueDate: formatDate(latestPayment.dueDate),
     settled: formatDate(settlement.settlementDate),
     calculated: formatDate(calculation.calculated),
-    value: convertToPounds(Math.trunc(paymentRequest.value / 4))
+    value: convertToPounds(latestPayment.value),
+    period: latestPayment.period

```

Comment:
These two values are the end goal of this whole PR!

calculating value of payment by dividing the annual value by 4 only works for quarterly and does not work if two instalments are settled at the same time.

The period was originally determined in `ffc-pay-statement-generator` but it does not have enough data to do this accurately.  So moved that functionality here so it is valid.